Sources/SwiftProtobuf/StringUtils.swift

```diff
@@ -20,6 +20,41 @@
 
 import Foundation
 
+/*
+  Note: Once our minimum support version is at least Swift 5.3, we
```

Comment:
Why wait until the min is that, can't you use a `#if swift(>= 5.3)` block with two code paths?

Reply:
I'll look into this later.